scss/_patterns_switch.scss

```diff
@@ -35,7 +35,7 @@ $knob-size: $sp-unit * 2;
   }
 
   .p-switch__slider {
-    background: $color-mid-light;
+    background: $colors--theme--background-neutral-default;
     border-radius: $knob-size;
     box-shadow: inset 0 2px 5px 0 transparentize($color-dark, 0.8);
```

Comment:
@bartaz Will leaving this as `$color-dark` (legacy generic color variable) cause any issues with the tokens work that we will be doing soon?

Reply:
what si background-neutral? $color-dark should be deprecated and replaced by $color-x-dark (pure black) I think.